agent/assets/registries_conf.j2

```diff
@@ -0,0 +1,69 @@
+{# base oc mirror registries.conf #}
```

Comment:
This would used in the disconnected ZTP scenario the most. Maybe I can move it there. It would also be needed if an install-config leaves out the ICSP section too.